atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/sweep/BackgroundSweeperImpl.java

```diff
@@ -204,12 +204,14 @@ static int adjustBatchParameter(int parameterValue) {
                         Optional<SweepProgress> progress = specificTableSweeper.getSweepProgressStore().loadProgress(
                                 tx);
                         if (progress.isPresent()) {
+                            log.info("Sweeping another batch of table: {}.",
```

Comment:
This will be printed too frequently. Looking at "started a table" gives us the information about which table is being swept. This doesnt seem to give any extra info. We could either log start row here or get rid of it?

Reply:
Agreed that it's probably be logged a lot. But I'd error on logging more than needed here, given the need we've recently seen for sweep. Also logging the batch startRow here.